src/redash_stmo/data_sources/link/extension.py

```diff
@@ -46,7 +45,7 @@ def get(self, data_source_id):
         try:
             result = {
                 "type_name": data_source.query_runner.name(),
-                "doc_url": data_source.options.get("doc_url", None)
+                "doc_url": DATASOURCE_URLS[data_source.query_runner.type()]
```

Comment:
I think we should still use getitem here to not always return a KeyError to the frontend by the generic exception handling below.

```suggestion
                "doc_url": DATASOURCE_URLS.get(data_source.query_runner.type(), None)
```

I assumed we'd also want to add the ability to have an additional mapping e.g. `DATASOURCE_INSTANCE_URLS` that map data source instance names (`data_source.name` here) to custom URLs? E.g. https://sql.telemetry.mozilla.org/data_sources/63 has a non-default documentation URL linking to the [standard SQL docs](https://cloud.google.com/bigquery/docs/reference/standard-sql/).

So something like

```python
DATASOURCE_INSTANCE_URLS = {
    "BigQuery (Beta)": "https://cloud.google.com/bigquery/docs/reference/standard-sql/",
}
```

and then extending the lookup to not only use `DATASOURCE_URLS` but also `DATASOURCE_INSTANCE_URLS`?